src/node.rs

```diff
@@ -57,6 +57,7 @@ use lightning_invoice::{
     Invoice,
 };
 use log::{debug, error, info, trace};
+use secp256k1::rand;
```

Comment:
I don't think we can use this? `getrandom` is what we do for entropy, is this working? Should probably have a test that covers initing a `node` with a vec of LSPs.

Reply:
Yeah it worked for me, it uses getrandom for randomness if you have the dep